src/main/java/com/gtnewhorizons/angelica/mixins/early/angelica/optimizations/MixinRendererLivingEntity.java

```diff
@@ -0,0 +1,49 @@
+package com.gtnewhorizons.angelica.mixins.early.angelica.optimizations;
+
+import com.gtnewhorizons.angelica.glsm.GLStateManager;
+import net.minecraft.client.renderer.entity.RendererLivingEntity;
+import net.minecraft.entity.EntityLiving;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.EnumChatFormatting;
+import net.minecraft.util.MathHelper;
+import org.lwjgl.opengl.GL11;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.Shadow;
+
+@Mixin(RendererLivingEntity.class)
+public abstract class MixinRendererLivingEntity {
+
+    @Shadow protected abstract float getDeathMaxRotation(EntityLivingBase entity);
+
+    /**
+     * @author mitchej123
+     * @reason The vanilla method triggers way too many allocations for no good reason
+     */
+    @Overwrite
```

Comment:
Did an overwrite as opposed to an inject because I didn't want to allocate a CancelableCallbackInfo

Reply:
Is `extends Render` needed or it's fine?

Reply:
Extends is rarely needed unless you're trying to cast something to it, and even then you can do an (Cls) (Object) style cast.

Tested and it works at least in so far as it no longer shows up on the memory profiler...

Reply:
![2024-01-27_15 59 56](https://github.com/GTNewHorizons/Angelica/assets/1894689/122ed2aa-cb7f-423e-a09c-244ce92ddb6a)